Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -75,6 +72,63 @@ configuration:
   name: latest_fetch_time_fieldname
   required: false
   type: 0
+- display: Extract Fields - CSV fields that will be parsed out of _raw notable events
+  name: extractFields
+  required: false
+  type: 12
+- additionalinfo: Used only for Mapping with the Select Schema option. The name of
+    the field that contains the type of the event or alert. The default value is "source",
+    which is a good option for Notable Events, however you may choose any custom field
+    that suits the need.
+  defaultvalue: source
+  display: Event Type Field
+  hidden: false
+  name: type_field
+  required: false
+  type: 0 
+- defaultvalue: 'false'
+  display: Use CIM Schemas for Mapping
+  name: use_cim
+  required: false
+  type: 8
+  additionalinfo: If selected, when creating a mapper using the `Select Schema` feature (Supported from XSOAR V6.0), Splunk CIM field will be pulled. See https://docs.splunk.com/Documentation/CIM/4.18.0/User/Overview for more info.
+- defaultvalue: None
+  additionalinfo: 'Choose the direction to mirror the incident: Incoming (from Splunk
+      to XSOAR), Outgoing (from XSOAR to Splunk), or Incoming and Outgoing (from/to
+      XSOAR and Splunk).'
+  display: Incident Mirroring Direction
+  hidden: false
+  name: mirror_direction
+  options:
+    - None
+    - Incoming
+    - Outgoing
+    - Incoming And Outgoing
+  required: false
+  type: 15
```

Comment:
If I recall correctly Ido Shavit has asked that this be placed on the top next to the mapper selection - should consult with him about this.